src/pages/drift/components/StartGRegulering.tsx

```diff
@@ -0,0 +1,69 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { Alert, Button, GuidePanel, Heading, Loader } from '@navikt/ds-react';
+import React, { useEffect, useState } from 'react';
+
+import { startRegulering } from '~api/reguleringApi';
+import * as reguleringApi from '~api/reguleringApi';
+import DatePicker from '~components/datePicker/DatePicker';
+import { pipe } from '~lib/fp';
+import { useApiCall } from '~lib/hooks';
+import { Nullable } from '~lib/types';
+import { toStringDateOrNull } from '~utils/date/dateUtils';
+
+import styles from '../index.module.less';
+
+const StartGRegulering = () => {
```

Comment:
Det er vel i utgangspunktet så generisk nå at det kan støtte andre typer regulering enn bare G-reguleringen, så mulig det burde gjenspeiles i navn rundt omkring også (også bare for å slippe `GRxxx`).